### PR TITLE
[BackgroundsV2 Project] [MODULAR] Removes Old Unused Culture Code

### PR DESCRIPTION
**Pull request by RimiNosha**
## About The Pull Request

Dead code, dead lore. Good bye!

Once this is merged, I will make the original PR (#18227) point to the backgrounds project branch instead.

Also, please don't squash merge any of these PRs that are going into the project branch, It will cause insane headaches for all involved if they are, cause I'm doing like 4 branches at once while PRing one due to how the code is bundled.

## How This Contributes To The Skyrat Roleplay Experience

I think this is self explanatory.

## Proof of Testing

Ded code gone, nothing to say other than it compiles on my machine<sup>:tm:</sup>.

## Changelog

No CL, not player facing. Hell, barely even admin or dev facing.

### CODE REVIEW

**Comment by Zonespace27** [comment]
Even if this doesn't affect GBP i'm setting it as a standard for other project PRs

**Comment by RimiNosha** [comment]
I'll tag it as no GBP in the title from now on to save you a couple of clicks.

File: modular_skyrat/modules/customization/modules/culture/culture/_culture.dm [hunk 1]
```diff
@@ -1 +0,0 @@
-/datum/cultural_info/culture
```
Comment (Zonespace27) [inline modular_skyrat/modules/customization/modules/culture/culture/_culture.dm:1]: lmao

Comment (RimiNosha) [reply modular_skyrat/modules/customization/modules/culture/culture/_culture.dm:1]: Some of the lore entries are gold, it's hilarious

**Pull request merged**